codepropertygraph/src/main/scala/io/shiftleft/passes/CpgPass.scala

```diff
@@ -81,6 +83,15 @@ abstract class CpgPass(cpg: Cpg, outName: String = "", keyPool: Option[KeyPool]
     }
   }
 
+  override def runWithBuilder(builder: BatchedUpdate.DiffGraphBuilder): Int = {
```

Comment:
Can you add a comment here that describes what the method does and what it returns?

Reply:
Added a comment to the API definition in CpgPassBase